plasma_framework/contracts/poc/fast_exits/Liquidity.sol

```diff
@@ -167,12 +166,12 @@ contract Liquidity is ERC721Full {
      * @dev Get Amount from contract after exit is processed - (to be updated)
      * @param exitId The exit id
     */
-    function getWithdrawal(uint160 exitId) public {
+    function getWithdrawal(uint168 exitId) public {
         require(
             super.ownerOf(exitId) == msg.sender,
             "Only the NFT owner of the respective exit can get the withdrawal"
         );
-        uint160[] memory exitIdList = new uint160[](1);
+        uint168[] memory exitIdList = new uint168[](1);
```

Comment:
I know it's not part of your task, but that was a great opportunity to rename of this variable to `exitIds` :+)